ytmusicapi/parsers/search.py

```diff
@@ -257,17 +258,25 @@ def _get_param2(filter):
     return filter_params[filter]
 
 
-def parse_search_suggestions(results, detailed_runs):
+def parse_search_suggestions(results, detailed_runs, feedback_tokens):
     if not results.get("contents", [{}])[0].get("searchSuggestionsSectionRenderer", {}).get("contents", []):
         return []
 
     raw_suggestions = results["contents"][0]["searchSuggestionsSectionRenderer"]["contents"]
     suggestions = []
 
+    count = 1  # Used for deleting a search suggestion
     for raw_suggestion in raw_suggestions:
```

Comment:
```suggestion
    for index, raw_suggestion in enumerate(raw_suggestions):
```

idiomatic Python. Remove the count variable